devel/carthage/Portfile

```diff
@@ -5,7 +5,7 @@ PortGroup           github 1.0
 PortGroup           xcodeversion 1.0
 
 use_xcode           yes
-github.setup        Carthage Carthage 0.34.0
+github.setup        Carthage Carthage 0.35.0
```

Comment:
The build currently fails for Xcode 12 (which has Swift 5.3). It is an issue already resolved upstream. In fact, the upstream commit addressing it (Carthage/Carthage@a33d3483b3) is the commit immediately after the 0.35.0 release (Carthage/Carthage@c7550f832f). In this case I personally would suggest using:
```
#github.setup        Carthage Carthage 0.35.0
# a33d3483b3 is commit immediately after 0.35.0 tag with fix for Xcode 12/Swift 5.3
github.setup        Carthage Carthage a33d3483b3
version             0.35.0
```
although others might insist on using a patchfile.

Reply:
I now notice there might be even more errors for Xcode 12 not yet resolved upstream: Carthage/Carthage#3019. I don't think this PR is expected to address those issues.

Reply:
Thank you for your kind advice.
As for https://github.com/macports/macports-ports/pull/8408#discussion_r487451496, resolved in https://github.com/macports/macports-ports/pull/8408/commits/86739707f74a994c16f072057c2b8712d5ef830a.
Then for https://github.com/macports/macports-ports/pull/8408#discussion_r487452720, yes I don't address all Xcode12 issues because it's still in beta.